examples/missing_values.py

```diff
@@ -18,6 +18,7 @@
   Score with the entire dataset = 0.56
```

Comment:
Could you please just make this a bit clearer by explaining above that in this example we artificially, randomly mark elements in a complete dataset as missing. I think "complete" might be better in this line than "entire".